src/sentry/middleware/integrations/parsers/msteams.py

```diff
@@ -95,6 +111,9 @@ def get_response(self) -> HttpResponseBase:
             logger.info("%s.no_regions", self.provider, extra={"path": self.request.path})
             return self.get_response_from_control_silo()
 
+        if self._check_if_event_should_be_sync(data=self.request_data):
+            return self.get_default_missing_integration_response()
```

Comment:
Interesting, so we don't handle synchronous events on our end at all? I'm also curious if MSTeams does anything if too many 4xx or 5xx errors are surfaced. Some other integrations like gitlab will eventually disable the webhook integration automatically if this occurs.

Reply:
We handle them synchronously if the integration cannot be inferred or if the organization does not have regions. Basically anything that calls `get_response_from_control_silo` is the synchronous method

Reply:
Just noticed the function i'm calling is wrong for the response lol